boards/arc/em_starterkit/em_starterkit.dts

```diff
@@ -27,6 +27,14 @@
 		zephyr,shell-uart = &uart1;
 	};
 
+	feature {
```

Comment:
maybe `zephyr,test-feature`?

Also, we should add a comment about what `feature is about`.

Reply:
Was meant to be `fixture`, not `feature`.  Everywhere.  Coffee must not have worked this morning.

Reply:
> Was meant to be fixture, not feature. Everywhere. Coffee must not have worked this morning.

In that case, could you update the title and PR description as well ?
It's quite confusing to read it, to just later discover this message which makes things clear.

Reply:
done; sorry for the confusion.